templates/fstab.j2

```diff
@@ -0,0 +1,16 @@
+# {{ ansible_managed }}
+{% set fmtstr='%-50s%-20s%-8s%-17s%s' -%}
+{{ fmtstr|format('# <filesystem>', '<mountpoint>', '<type>', '<options>', '<dump/pass>') }}
+
+{%- set mount_options='remount,rw' -%}
+{%- if vm.storage_type|default(xen_vman_storage_type) == "iscsi" -%}
+	{%- set root_device='/dev/xvda' -%}
+	{%- set filesystem=vm.filesystem|default(xen_vman_default_filesystem) -%}
+	{%- set pass_str='0 1' -%}
+{%- elif vm.storage_type|default(xen_vman_storage_type) == "nfs" -%}
```

Comment:
what about additionally defined filesystems?

Reply:
This configuration should only get the system to work. All future configurations should me made by a separated role.